build.fsx

```diff
@@ -8,51 +8,39 @@ open Fake.EnvironmentHelper
 open Fake.FscHelper
 open Fake.ProcessHelper
 
+#load "project.fsx"
+
+let ensureProjectLoaded() =
+    tracefn "Project definition loaded for directory: %s" project.BaseDir
```

Comment:
This currently causes an error b/c the `project` value does not appear to have imported from `project.fsx`.